Formula/gstreamer.rb

```diff
@@ -46,14 +52,11 @@ def install
     inreplace "meson.build",
       "cdata.set_quoted('PLUGINDIR', join_paths(get_option('prefix'), get_option('libdir'), 'gstreamer-1.0'))",
       "cdata.set_quoted('PLUGINDIR', '#{HOMEBREW_PREFIX}/lib/gstreamer-1.0')"
+    inreplace "gst/gstregistry.c", " defined(HAVE_DLADDR)", " 0"
```

Comment:
Temporary workaround to restore `PLUGINDIR` lookup (existing inreplace). Don't have time to look into merging formulae right now. Maybe if I have time later on.